easybuild/easyconfigs/b/bcl-convert/bcl-convert-4.0.3.eb

```
@@ -0,0 +1,19 @@
+easyblock = 'Rpm'
+
+name = 'bcl-convert'
+version = '4.0.3'
+homepage = 'https://www.illumina.com/products/by-type/informatics-products/basespace-sequence-hub/apps/bcl-convert.html'
+description = """The BCL Convert App generates demultiplexed FASTQ files from a run as input. The core algorithm is BCL Convert.
+
+for more information: https://www.illumina.com/products/by-type/informatics-products/basespace-sequence-hub/apps/bcl-convert.html
+"""
+toolchain = {'name': 'system', 'version': ''}
+
+sources = ["https://webdata.illumina.com/downloads/software/bcl-convert/bcl-convert-4.0.3-2.el7.x86_64.rpm"]
+checksums = ["be4f3d1d31cb36618f909cdbc0ed0fe70ef57a645e46b421b6e1a7f90c6c9430"]
+
+sanity_check_commands = ["%(name)s --version"]
+sanity_check_paths = {
+    'files': ["usr/bin/bcl-convert"],
+    'dirs': []
+}
```

Please add the missing `moduleclass`:

```suggestion
}

moduleclass = 'bio'
```